### PR TITLE
ENH add bot automerge command

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [ ] Pushed the branch to main repo
* [ ] CI passed on the branch

Matt's Checklist:
* [x] needs tests
* [x] will need https://github.com/conda-forge/conda-smithy/pull/1217 and a new release of conda-smithy.

### CODE REVIEW

File: conda_forge_webservices/commands.py [hunk 1]
```diff
@@ -285,6 +302,34 @@ def issue_comment(org_name, repo_name, issue_num, title, comment):
...
 
+def add_bot_automerge(repo):
+    # copy in the workflow def from smithy
+    from conda_smithy.configure_feedstock import conda_forge_content
+
+    workflows_dir = os.path.join(repo.working_dir, ".github", "workflows")
+    os.makedirs(workflows_dir, exist_ok=True)
+    dest_main_yml = os.path.join(workflows_dir, "main.yml")
+    src_main_yml = os.path.join(
+        conda_forge_content, "templates"", main.yml.tmpl")
```
Comment (CJ-Wright) [inline conda_forge_webservices/commands.py]:
```suggestion
        conda_forge_content, "templates", main.yml.tmpl")
```

Comment (beckermr) [reply conda_forge_webservices/commands.py]: good catch!

**Comment by CJ-Wright** [comment]
I can cut a smithy release once the PR is merged

File: conda_forge_webservices/commands.py [hunk 2]
```diff
@@ -285,6 +302,34 @@ def issue_comment(org_name, repo_name, issue_num, title, comment):
...
+    dest_main_yml = os.path.join(workflows_dir, "main.yml")
+    src_main_yml = os.path.join(
+        conda_forge_content, "templates", "main.yml.tmpl")
+    shutil.copyfile(src_main_yml, dest_main_yml)
+
+    # now add to conda-forge.yml
+    cf_yml = os.path.join(repo.working_dir, "conda-forge.yml")
+    yaml = YAML()
+    with open(cf_yml, 'r') as fp:
+        cfg = yaml.load(fp)
```
Comment (beckermr) [inline conda_forge_webservices/commands.py]: make sure they have a file

**Comment by beckermr** [comment]
closing in favor of #292

**Pull request closed without merging**